src/App.tsx

```diff
@@ -1,25 +1,46 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [arrItems, setArrayItems] = useState<string[]>([]);
```

Comment:
The 'arrItems' state seems to be a duplicate of the 'items' array. Since you're setting 'arrItems' to 'items' directly inside a useEffect and not modifying it elsewhere, you could directly use 'items' and remove the 'arrItems' state to simplify your component.